lib/cli/interactive-setup/index.js

```diff
@@ -63,6 +66,21 @@ module.exports = async (context) => {
     }
   }
 
+  if (context.isConsole && options.dev && initialContext.isInServiceContext) {
+    const compatibilityMap = await apiRequest('/api/inventories/compatibility', {
+      method: 'GET',
+    });
+    const devModeRuntimeCompatibility = compatibilityMap.mode.dev.runtimes;
+    const { provider } = context.serverless.service;
+    if (!devModeRuntimeCompatibility.includes(provider.runtime)) {
+      context.incompatibleRuntime = true;
+      log.error('This services runtime is not currently supported by Serverless Console Dev Mode.');
+      throw new Error('INCOMPATIBLE_RUNTIME');
```

Comment:
Do we really want to throw here?

Even if, we should use `ServerlessError` and configure it with human friendly message which will be displayed to user

Reply:
This is a good question. It wasn’t clear to me if there was a way for me to end the execution early without throwing. If I put a return here it threw a lint error 🤔

Reply:
Question, is do we want here (1) abort the command and present user with the error message then we should throw (but as mentioned above using dedicated interface and with human friendly message)
Or do we want to  (2) just show error stating that given step cannot be accomplish and let command follow with other steps (as e.g. consol integration). In this case we should just log the error, do not set `context.isConsoleDevMode: true`  and let process continue

Reply:
🤔 I guess it might be fine to just let the process continue and just log the error. That would probably be the least inconvenient.